### DISCUSSION TITLE
Transaction confirmation delay

### DISCUSSION THREAD

**Discussion by Knightfall11** [jumperexchange/jumper-exchange#841] [Ideas]
BUG/UI: my transaction is already 77 seconds in my account before jumper says completed. Update your onchain checker so people get almost instant confirmation instead of waiting 77 seconds to get a confirmation while the funds are already there, big user experience improvement.

**Comment by chybisov** [comment]
@Knightfall11 We've made significant improvements over the past few months. Please give it another try - we think you'll notice the difference 😉
